tool/README.md

```diff
@@ -24,29 +24,28 @@ Make sure:
 
 ### Prepare the release
 
-#### Update the DevTools version number
-
-- Make sure your working branch is clean
-- Run the `tool/release_helper.sh` script with `minor` or `major`.
-   `./tool/release_helper.sh [minor|major]`
-- This creates 2 branches for you:
-    - Release Branch
-    - Next Branch
-- The following steps will guide you through how these branches will be prepared and merged.
-- **For your convenience, the `tool/release_helper.sh` script exports the following two variables to the terminal it is run in:**
-  - `$DEVTOOLS_RELEASE_BRANCH`
-  - `$DEVTOOLS_NEXT_BRANCH`
-
-#### Verify the version changes for `$DEVTOOLS_RELEASE_BRANCH`
-
-Verify release_helper.sh script:
+#### Ensure you have Github CLI installed
+- If you need to install the Github CLI you can run:
+    `brew install gh`
+- More information can be found at: https://cli.github.com/manual/installation
```

Comment:
can we put this in a note in the markdown instead of its own section? idk what the proper term for this is, but it should look more like an alert than a section